ktoolbox/k8sClient.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, kubeconfig: typing.Optional[str] = None):
     @staticmethod
     def _get_oc_cmd(cmd: Union[str, Iterable[str]]) -> list[str]:
         if isinstance(cmd, str):
-            return shlex.split(cmd)
+            return ["bash", "-c", cmd]
```

Comment:
note that `host.run()` uses `("/bin/sh", "-c", cmd)` (see `_cmd_to_argv()`).

I feel that is what we usually expect when we run a shell script. That we find some posix-compatible script at `/bin/sh`.


in general, which shell script interprets the command depends on what we find installed in the the host (in case of `LocalHost.run()` and `RemoteHost.run()`) or the container image (in case of `K8sClient.oc_exec()`). So, it is not generally portable beyond the most basic assumption that this is hopefully a Posix shell.

Point being: can we use `return ["/bin/sh", "-c", cmd]` ?